README.md

```diff
@@ -31,26 +14,14 @@ The following algorithms will be implemented from scratch in Rust:
 - First-In-First-Out
 - Single-Random-Draw
```

Comment:
Shouldn't we add bnb, knapsack and others too here?

Reply:
those are already implemented, so should change the wording as well

Reply:
' Other algorithms implemented : '